meerevm/amana/config.go

```diff
@@ -114,6 +114,13 @@ func Genesis(genesisFile string) (*core.Genesis, error) {
 		}
 		return genesis, nil
 	}
+
+	// TODO:Purely for compatibility with the testnet network, it can be completely removed if recreated genesis in the future
```

Comment:
can we create a separted json file `deprecated_amana.json` for the old genesis which only support  `London block` and let the latest code works with` Cancun` by default?

Reply:
https://github.com/Qitmeer/qng/pull/775/commits/197a1c81901eae2f53fee9f4a4c763697d3aab3a